pages/enroll/enroll.sql

```diff
@@ -46,3 +46,14 @@ WHERE
     AND e.course_instance_id = $course_instance_id
     AND check_course_instance_access($course_instance_id, u.uid, u.institution_id, $req_date)
 RETURNING e.id;
+
+-- BLOCK lti_course_instance_lookup
+SELECT
+    ci.short_name AS ci_short_name, ci.long_name AS ci_long_name,
+    plc.short_name AS plc_short_name, plc.title
```

Comment:
AFAICT, only these two are used on the LTI page.

```suggestion
    plc.short_name AS plc_short_name, ci.long_name AS ci_long_name
```